File: lib-cli/src/workflow/render/producer/rust/static/events/connected.rs
use super::{identification, producer::Control, Context};
use fiber::server;

#[allow(unused_variables)]
pub async fn emit<E: std::error::Error, C: server::Control<E> + Send + Clone>(
    identification: &mut identification::Identification,
    filter: &identification::Filter,
    context: &mut Context,
    control: &Control<E, C>,
) -> Result<(), String> {
    Err(String::from(
        "Event emitter \"connected\" isn't implemented",
    ))
}

File: lib-cli/tmp/server-ts/src/producer/events/connected.ts
import * as Protocol from "implementation/protocol";
import {
	Identification,
	Filter,
	Producer,
	Context,
} from "implementation/events";

export function emit(
	consumer: Identification,
	filter: Filter,
	context: Context,
	producer: Producer<Context>
): Promise<void> {
	return Promise.reject(
		new Error(`Handler for event "connected" isn't implemented`)
	);
}
